### ISSUE TITLE
add divider to top of first item in LinearLayoutManager

### ISSUE THREAD

**Issue by TikTak123** [fondesa/recycler-view-divider#68]
Add please divider to top of first item in LinearLayoutManager like in this gist
https://gist.github.com/lapastillaroja/858caf1a82791b6c1a36

One of my project has design where I need this feature. Maybe this is usefull for other also.

**Comment by fondesa** [comment]
Since there are other users who asked this, even if I don't agree completely to add this feature inside the library (the top divider can be represented as an external view since it doesn't "divide" anything), I'll add it anyway. 
It will be implemented in the version [4.x](https://github.com/fondesa/recycler-view-divider/milestone/2) .

**Comment by fondesa** [comment]
Just released the version [3.2.0](https://github.com/fondesa/recycler-view-divider/releases/tag/3.2.0) which contains `showFirstDivider` to show the top divider.

**e.g.** (kotlin)

```kotlin
context.dividerBuilder()
    .showFirstDivider()
    .build()
    .addTo(recyclerView)
```
